toast-tk-runtime/src/main/java/io/toast/tk/runtime/parse/FileHelper.java

```diff
@@ -44,12 +52,115 @@ private FileHelper() {
 	public static InputStream getInputStream(final String filename) {
 		LOG.debug("Open input stream: " + filename);
 		String fullFileName = filename;
-		if (!filename.startsWith("\\")) {
+		if (!(filename.startsWith("\\") || filename.startsWith("C:"))) {
 			fullFileName = "/" + fullFileName;
 		}
 		return FileHelper.class.getResourceAsStream(fullFileName);
 	}
 
+	public static String readFile(String fileName) {
+		String lines = "";
+		try {
+			lines = new String(Files.readAllBytes(Paths.get(fileName)), StandardCharsets.UTF_8);
+			return lines;
+		} 
+        catch(IOException ex) {
+        	String exception = "Error reading file '" + fileName + "' caused by " + ex.getMessage();
+        	LOG.info(exception);
+            return exception;       
+        }
+    }
+
+	public static String getStringFromInputStream(InputStream is) {
+		BufferedReader br = null;
+		StringBuilder sb = new StringBuilder();
+		
+		String line;
+		try {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Change this "try" to a try-with-resources. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2093)